datarepo-clienttests/src/main/java/testscripts/BulkLoad.java

```diff
@@ -31,26 +34,25 @@ public void setParameters(List<String> parameters) throws Exception {
     }
   }
 
-  public void setup(Map<String, ApiClient> apiClients) throws Exception {
-    bulkLoadUtils = new BulkLoadUtils();
-    bulkLoadUtils.bulkLoadSetup(apiClients, billingAccount);
-  }
-
   // The purpose of this test is to measure scaling of bulk load.
-  public void userJourney(ApiClient apiClient) throws Exception {
+  public void userJourney(TestUserSpecification testUser) throws Exception {
+    ApiClient apiClient = DataRepoUtils.getClientForTestUser(testUser, server);
     RepositoryApi repositoryApi = new RepositoryApi(apiClient);
 
     // set up and start bulk load job
-    BulkLoadArrayRequestModel arrayLoad = bulkLoadUtils.buildBulkLoadFileRequest(filesToLoad);
+    BulkLoadArrayRequestModel arrayLoad =
+        BulkLoadUtils.buildBulkLoadFileRequest(
+            filesToLoad, billingProfileModel.getId(), datasetSummaryModel.getId());
     JobModel bulkLoadArrayJobResponse =
-        repositoryApi.bulkFileLoadArray(bulkLoadUtils.getDatasetId(), arrayLoad);
+        repositoryApi.bulkFileLoadArray(datasetSummaryModel.getId(), arrayLoad);
 
-    // wait for the job to complete and print out results
-    bulkLoadUtils.getAndDisplayResults(repositoryApi, bulkLoadArrayJobResponse);
-  }
+    // wait for the job to complete and print out results to debug log level
+    BulkLoadResultModel loadSummary =
+        BulkLoadUtils.getAndDisplayResults(repositoryApi, bulkLoadArrayJobResponse);
 
-  public void cleanup(Map<String, ApiClient> apiClients) throws Exception {
-    KubernetesClientUtils.changeReplicaSetSizeAndWait(1);
```

Comment:
Do we need to reset the replica set size?

Reply:
I think we don't need to because we already set the replica set size at the beginning of every test run. There's no real reason to "put things back to normal" because we wipe out the deployment before every test run anyway.